### ISSUE TITLE
Lab 5-1: KimTay Pet Supplies Task 6 Expected Answer Issue

### ISSUE THREAD

**Issue by dpaxson1** [microsoft/vscode#207596]
I believe the expected answer for Task 6 to be wrong.

+---------+------------+-----------+
| CUST_ID | FIRST_NAME | LAST_NAME |
+---------+------------+-----------+
|   294   |  Samantha  |   Smith   |
|   182   |   Billy    |  Rufton   |
|   435   |   James    | Gonzalez  |
|   125   |    Joey    |   Smith   |
|   435   |   James    | Gonzalez  |
|   616   |   Sally    |   Cruz    |
+---------+------------+-----------+

Should be:

┌─────────┬────────────┬───────────┐
│ CUST_ID │ FIRST_NAME │ LAST_NAME │
├─────────┼────────────┼───────────┤
│ 294     │ Samantha   │ Smith     │
├─────────┼────────────┼───────────┤
│ 182     │ Billy      │ Rufton    │
├─────────┼────────────┼───────────┤
│ 435     │ James      │ Gonzalez  │
├─────────┼────────────┼───────────┤
│ 125     │ Joey       │ Smith     │
├─────────┼────────────┼───────────┤
│ 616     │ Sally      │ Cruz      │
└─────────┴────────────┴───────────┘

Basically, 435 James Gonzalez should not repeat.

I used:
SELECT CUST_ID, FIRST_NAME, LAST_NAME
FROM CUSTOMER
WHERE CUST_ID IN (SELECT CUST_ID
                        FROM INVOICES
                            WHERE INVOICE_DATE <> "2021-11-15");

Version: 1.85.2
Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/119.0
Embedder: codespaces

**Comment by gjsjohnmurray** [comment]
/question please use your course support channel
